fe1-web/jest/setup.js

```diff
@@ -18,3 +18,20 @@ jest.mock('react-native-reanimated', () => {
 // Silence the warning: Animated: `useNativeDriver` is not supported
 // because the native animated module is missing
 jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');
+
+// Icons and snapshot tests do not work nice together
+// See https://github.com/expo/expo/issues/3566
+jest.mock('@expo/vector-icons');
+
+// qr codes are svgs and thus generate huge snapshot files
+jest.mock('core/components/QRCode.tsx', () => 'qrcode');
+
+// make functions return a value independent of the CI locale
+jest.spyOn(Date.prototype, 'toLocaleDateString').mockReturnValue('2022-05-28');
+jest.spyOn(Date.prototype, 'toLocaleTimeString').mockReturnValue('00:00:00');
+
+const FIXED_SYSTEM_TIME = new Date(1620255600 * 1000); // 5 May 2021
+
+jest.useFakeTimers('modern');
+jest.setSystemTime(FIXED_SYSTEM_TIME);
+jest.useRealTimers();
```

Comment:
Does it work this way? I feel like `useRealTimers()` would be executed before the tests.

Reply:
yeah no it seems something is off since the snapshot tests seem to have been reverted again 🤔 I just think it would be nice to have the same fake time for all tests but I am not sure how to achieve this 😅 I guess I should look at the docs a bit and try to find a way later to make this work but I will go on holidays for 4 weeks tomorrow morning:)

Reply:
I completely agree with this, having to set the same system time in every test is repetitive. Setting the time before all tests would be better.
Enjoy your holidays!